components/Templates/includes/auto-template/Pods_Templates_Auto_Template_Front_End.php

```diff
@@ -155,44 +181,54 @@ public function auto_pods() {
 			// get possible pods
 			$the_pods = $this->the_pods();
 
-			// start output array empty
+			// start output array empty.
 			$auto_pods = array();
 
-			// get pods api class
+			// get pods api class.
 			$api = pods_api();
 
-			// loop through each to see if auto templates is enabled
+			// loop through each to see if auto templates is enabled.
 			foreach ( $the_pods as $the_pod => $the_pod_label ) {
 				// get this Pods' data.
 				$pod_data = $api->load_pod( array( 'name' => $the_pod ) );
+				$options  = pods_v( 'options', $pod_data, array() );
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 7a86ddd5ec77aeb6c75af7853a4b2f63</sub>